src/Microsoft.ML.Data/DataView/TypedCursor.cs

```diff
@@ -217,6 +217,90 @@ public RowCursor<TRow>[] GetCursorSet(Func<int, bool> additionalColumnsPredicate
                  .ToArray();
         }
 
+        public static void ValidateMemberInfo(MemberInfo memberInfo, IDataView data)
+        {
+            if (!SchemaDefinition.CheckMemberInfo(memberInfo))
+                return;
```

Comment:
Is this not an error condition? Should you be throwing something here?

Reply:
I don't think it should throw on false. The CheckMemberInfo already throws exception when appropiate. And if it returns false, it only means we shouldn't look more into the member. So i think the logic here is fine.

---
In reply to: [431431962](https://github.com/dotnet/machinelearning/pull/5114#discussion_r431431962) [](ancestors = 431431962)